### ISSUE TITLE
12885491: Missing accessiblityIdentifier on UIBarItem class

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#5441]
#### Description

UIBarItem class does not have a public 'accessibilityIdentifier' property.
## 

Product Version: iOS 6.01
Created: 2012-12-14T21:05:02.599812
Originated: 2012-12-14T14:49:00
Open Radar Link: http://www.openradar.me/12885491
